storage/src/main/scala/ch/epfl/bluebrain/nexus/storage/files/CopyFileOutput.scala

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.bluebrain.nexus.storage.files
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.storage.{encJPath, encUriPath}
+import io.circe.Encoder
+import io.circe.generic.semiauto.deriveEncoder
+
+import java.nio.file.Path
+
+final case class CopyFileOutput(
+    sourcePath: Uri.Path,
+    destinationPath: Uri.Path,
+    absoluteSourceLocation: Path,
+    absoluteDestinationLocation: Path
```

Comment:
I would just want to double check this response information would be used by a client?

Reply:
Yeah I was trying to figure that out - it seems in delta we take the absolute path from the `location` field of a regular move / create remote file response. I wasn't sure whether it can be determined without it being returned? So I figured stick to the same pattern 🤷